source/blog/2016-11-02-ovirt-provider-ovn.md

```diff
@@ -119,6 +119,11 @@ Enable the service in order to make it start at boot, use:
 
     systemctl enable ovirt-provider-ovn
 
+OVN central must be configured to listen to requests on appropriate ports:
+
+    ovn-sbctl set-connection ptcp:6642
+    ovn-nbctl set-connection ptcp:6641
+
```

Comment:
This is on central, not hosts.
I will add a script for this, also add this to engine-setup.
But I think for now it should be on the blog.